objcgen/objcgenerator-helpers.cs

```diff
@@ -12,6 +12,29 @@ public partial class ObjCGenerator {
 
 		// get a name that is safe to use from ObjC code
 
+		public static Dictionary<string, string> ObjCTypeToArgument = new Dictionary<string, string> {
```

Comment:
I don't think this was merged when the PR go out initially but yes it makes sense, now, to move the code there.